### DISCUSSION TITLE
Direct bind between Aqara vibrations sensor and Sonoff ZBMINI ?

### DISCUSSION THREAD

**Discussion by Et3rnel** [Koenkk/zigbee2mqtt#16494] [Devices]
👋 Hello,
I'm trying to bind an Aqara vibrations sensor DJT11LM to trigger a Sonoff ZBMINI directly, from Zigbee2MQTT interface.

I cannot choose a cluster and the **Bind** button seems to be deactivated.

Do you know why and how I can check if it's both devices are compatible with direct binding?

Thanks

**Comment by Koenkk** [comment]
`DJT11LM` doesn't support this

**Reply by Et3rnel** [reply]
Alright that's what I thought, thanks for your answer.
